tardis/tests/tests_slow/plot_helpers.py

```diff
@@ -0,0 +1,61 @@
+import tempfile
+
+from pytest_html import extras
+import tardis
+
+
+thumbnail_html = """
+<div class="image" style="float: left">
+    <a href="#">
+        <img src= "{dokuwiki_url}lib/exe/fetch.php?media=plots:{githash}_{name}.png" />
+    </a>
+</div>
+"""
+
+
+class PlotUploader(object):
+    def __init__(self, request):
+        self.request = request
+        self._plots = list()
+        self.plot_html = list()
+        self.dokuwiki_url = self.request.config.dokureport.dokuwiki_url
+
+    def add(self, plot, name):
+        """
+        Accept a `matplotlib.pyplot.figure` and add it to self._plots.
```

Comment:
can you please do `numpydoc` style documentation. for all functions that have documentation.